### PR TITLE
Docs: Expression class docs

### PR DESCRIPTION
**Pull request by santouits**

### CODE REVIEW

File: doc/classes/Expression.xml [hunk 1]
```diff
@@ -19,18 +37,21 @@
...
 			</return>
 			<description>
+				Returns the error text if [method parse] has failed.
 			</description>
 		</method>
 		<method name="has_execute_failed" qualifiers="const">
 			<return type="bool">
 			</return>
 			<description>
+				Returns true if [method execute] has failed.
```
Comment (akien-mga) [inline doc/classes/Expression.xml]: Our convention is to use code markup for `true` and `false` when used as boolean values, so it should be `[code]true[/code]`.

**Comment by akien-mga** [comment]
Looks quite good, thanks! If you can do the requested changes with an amend (`git commit --amend` and then `git push --force`), it should be good to merge.

**Comment by Zireael07** [comment]
How about providing an expression example? I guess 2+2 is an expression, but what about (sin(x) *20)-atan(x, y) or some such?

File: doc/classes/Expression.xml [hunk 2]
```diff
@@ -1,8 +1,26 @@
...
+		A class that stores an expression you can execute.
 	</brief_description>
 	<description>
+		An expression can be made of any arithmetic operation, built-in math function call, method call of a passed instance, or built-in type construction call.
+		In the following example we use a [LineEdit] node to write our expression and show the result.
+		[codeblock]
+		onready var expression = Expression.new()
+
+		func _ready():
+		    $LineEdit.connect("text_entered", self, "on_text_entered")
```
Comment (Naryosha) [inline doc/classes/Expression.xml]: The binding method names don't match. Line 15 it's `_on_text_entered`.

**Comment by santouits** [comment]
replaced with [enum @GlobalScope.Error]
added [code] tag around the true
added expression example
corrected signal binding

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**